app/src/lib/editors/win32.ts

```diff
@@ -60,7 +70,8 @@ type WindowsExternalEditor = {
 
   /** Value of the Publisher registry key that belongs to this editor. */
   readonly publisher: string
-} & WindowsExternalEditorPathInfo
+} & WindowsExternalEditorPathInfo &
+  WindowsExternalEditorJetbrainsToolbox
```

Comment:
Since this is just a fiel you added, I don't think we need an additional type to `&` here. `WindowsExternalEditorPathInfo` was different because it has two different types inside.

However, I'm not sure about the new `toolboxShellScriptName` optional field because it's not tied/required for JetBrains IDEs, but I don't see an easy way of getting TypeScript to enforce that.

In any case, I think the field name should be more JetBrains specific, something like `jetBrainsToolboxScriptName`

Reply:
Initially, I thought it might require more fields, so I added the type. But in its final form, since it's a single and optional field, I added it to the `WindowsExternalEditor` type and renamed it as you suggested to `jetBrainsToolboxScriptName`